server/controllers/events/events_controller.go

```diff
@@ -566,8 +567,8 @@ func (e *VCSEventsController) handleCommentEvent(logger logging.SimpleLogging, b
 		}
 	}
 
-	// It's a comment we're gonna react to, so add a reaction.
-	if e.EmojiReaction != "" {
+	// It's a comment we're gonna react to (as long as emoji reactions aren't disabled), so add a reaction.
```

Comment:
```suggestion
	// It's a comment we're going to react to (as long as emoji reactions aren't disabled), so add a reaction.
```

Lets fix the slang whilst we are changing this.

Reply:
Ha, good one!